lib/main_common.pm

```diff
@@ -549,12 +549,6 @@ sub load_system_role_tests {
     }
 }
 sub load_jeos_tests {
-    if (get_var('PREPARE_RPI')) {
```

Comment:
I see that some test suites on osd still have this variable set. Could you please remove it there as well?

Reply:
Ah, you are right - it should be ok by now, I removed "convert_image-RPi" testsuite from osd. 

Thanks for the merge.